catalogue/__init__.py

```diff
@@ -149,7 +149,7 @@ def get_entry_point(self, name: str, default: Optional[Any] = None) -> Any:
 
     def _get_entry_points(self) -> List[importlib_metadata.EntryPoint]:
         if isinstance(AVAILABLE_ENTRY_POINTS, SelectableGroups):
-            return cast(AVAILABLE_ENTRY_POINTS, SelectableGroups).select(group=self.entry_point_namespace)
+            return cast(SelectableGroups, AVAILABLE_ENTRY_POINTS).select(group=self.entry_point_namespace)
```

Comment:
Ugh, I also screwed up the previous suggestion. The `cast` should not be necessary here.

Reply:
No worries, I removed it manually. It's weird that my PyCharm doesn't get the hint without the `cast` though. Guess that's more of a PyCharm issue.